### PR TITLE
SM-6323: Add new Activity fields to open data

### PR DESCRIPTION
**Pull request by caoimheb**

### CODE REVIEW

**Review by ccurran01 — APPROVED** [review]

File: docs/api-documentation/json/event-notifier-message.json [hunk 1]
```diff
@@ -281,7 +289,7 @@
           "object_data": {
             "required": [
               "activity_reference_number", "activity_coordinates", "activity_name", "start_date", "end_date", "activity_location_type",
-              "traffic_management_required", "collaborative_working", "cancelled"
+              "activity_location_description", "traffic_management_required", "collaborative_working", "cancelled"
```
Comment (Kellym-Kainos) [inline docs/api-documentation/json/event-notifier-message.json:292]: activity_type_details too?

Comment (caoimheb) [reply docs/api-documentation/json/event-notifier-message.json:292]: It's an optional field

Comment (Kellym-Kainos) [reply docs/api-documentation/json/event-notifier-message.json:292]: ah! should it be optional in our data models then?

Comment (caoimheb) [reply docs/api-documentation/json/event-notifier-message.json:292]: I was sure that I had made it optional in the data models, but looking back at the PRs that is obviously not the case! I'll update now...

**Pull request merged**